### PR TITLE
support for unstable SLOs in chronoctl

### PR DESCRIPTION
**Pull request by aschepis**

### CODE REVIEW

**Comment by aschepis** [comment]
This change is part of the following stack:

- #44 ◀

<sub>Change managed by [git-spice](https://abhinav.github.io/git-spice/).</sub>

File: tools/cmd/chronogen/generator.go [hunk 1]
```diff
@@ -113,9 +116,19 @@ func (e entitySpec) NameP() string {
 		// Singletons shouldn't have singular/plural applied.
 		return upperCamel(e.Entity.Name)
 	}
+	if e.Entity.Name == "slos" {
+		return "SLOs"
+	}
 	return inflect.Pluralize(upperCamel(e.Entity.Name))
 }
 
+func (e entitySpec) PayloadNameP() string {
```
Comment (prashantv) [inline tools/cmd/chronogen/generator.go:125]: doc comment that calls out where this is intended to be used

File: src/generated/cli/configv1/datasets.gen.go [hunk 2]
```diff
@@ -539,6 +539,9 @@ spec:
                       error:
                         # The value the filter compares to the target trace or span field.
                         value: <true|false>
+                      is_root_span:
```
Comment (prashantv) [inline src/generated/cli/configv1/datasets.gen.go:542]: need to call out changes in the configv1 specs in the changelog

File: Makefile [hunk 3]
```diff
@@ -12,7 +12,7 @@ GO_BUILD_COMMON_ENV       := CGO_ENABLED=0
 GO_RELEASER_WORKING_DIR   := /go/src/github.com/chronosphere/chronoctl
 GO_RELEASER_RELEASE_ARGS  ?= --clean
 
-UNSTABLE_ENTITIES := link-templates,saved-trace-searches,dashboards,trace-tail-sampling-rules,services,metric-usages-by-metric-name,metric-usages-by-label-name
+UNSTABLE_ENTITIES := link-templates,saved-trace-searches,dashboards,trace-tail-sampling-rules,services,metric-usages-by-metric-name,metric-usages-by-label-name,slos
```
Comment (prashantv) [inline Makefile:15]: call out new slo unstable entity in changelog

**Review by prashantv — APPROVED** [review]

**Pull request merged**